src/main/java/bimo/Bimo.java

```diff
@@ -0,0 +1,62 @@
+package bimo;
+import bimo.command.Command;
+
+/**
+ * Represents a chatbot class.
+ */
+public class Bimo {
+    public static final String NAME = "bimo";
```

Comment:
Great job on making the name of your chatbot 'final', as this is a variable that is not going to change